docs/source/model_design/risk_factors.rst

```diff
@@ -1,55 +1,299 @@
+measurement of the flow of new cases arising from the population is more informative. Incidence focuses on new cases. There are three main measures of incidence: **risk**, **rate**, and **odds**. 
+
+Risk
+****
+
+**Incidence risk**, also called *incidence proportion*, *attack rate*, or *cumulative incidence* is the probability of occurence of disease among a disease free, at risk, population during a specified time period. It is the number of new cases of disease during a defined period of time divided by the population at the start of the time period. Like any proportion, risk has no time units but the time period to which it applies must be specified, otherwise it is not interpretable. The survival proportion is 1 minus incidence proportion.      
+
++------------------------+-----------------------------------------------+
+|   Measure              | Math                                          |
++------------------------+-----------------------------------------------+
+|| Incidence risk        |:math:`\frac{\text{new cases of disease during |
+|| Incidence proportion  |time period}}{\text{disease free, at risk,     |
+|| Attack rate           |population at the start of the time period}}`  |
+|| Cumulative incidence  |                                               |
++------------------------+-----------------------------------------------+
+
+Example: 
+
+  * A group of 5000 healthy women aged 45–75 years was identified at the beginning of 1981 and followed up for five years. During this period, 20 new cases of breast cancer were  detected. Hence, the risk of developing breast cancer in this population during this five-year period was 20/5000 = 0.4%.
+  * A total of 13 264 lung cancer cases in males were diagnosed in a certain population in 1971. These cases were followed up for five years. At the end of this follow-up period, only 472 cases were still alive. The probability of surviving during this five-year period was 472/13 264 = 3.6%. Thus, the probability of dying during the period was 100% – 3.6% = 96.4%. 
+
+In the second example, the measures are risks, as they represent the proportion of lung cancer cases who were still alive (or who died) at the end of the follow-up period out of all cases diagnosed at the beginning of the study. These calculations assume that all individuals were followed up for the entire five-year period (or until death if it occurred earlier). These measures are often called survival and fatality ‘rates’; this is incorrect as, by definition, they are proportions. Risk is a measure commonly used to quantify the survival experience of a group of subjects
+
+Odds
+****
+
+Another measure of incidence is odds of disease, which is the ratio of the total number of cases to the total number of persons who remained disease free over the study period.

```

Comment:
Do you perhaps mean that another measure for risk comparison is incidence odds? I think odds is not inherently a measure of incidence, but I could definitely be incorrect in my understanding!

Reply:
changed it to measure of disease occurrence/frequency